docs/content/preview/admin/yb-admin.md

```diff
@@ -1840,6 +1840,38 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
 
 ---
 
+#### get_replication_status
+
+Returns the replication status of all consumer streams. If *producer_universe_uuid* is provided, this will only return streams that belong to an associated universe key.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> get_replication_status [ <producer_universe_uuid> ]
```

Comment:
Just for consistency...
```suggestion
./bin/yb-admin \
    -master_addresses <master-addresses> \
    get_replication_status [ <producer_universe_uuid> ]
```

Reply:
We have a mix between `./bin/` and empty prefixes all over this page for `yb-admin`. Do you mind if I go ahead and change them all to be consistent?

Reply:
The convention appears to be not to use the ./bin/ in the syntax, only in the Examples, and it appears to be consistent throughout, so I wouldn't change this...

Reply:
Thanks for pointing this out! Leave as-is for sure.